src/sql/src/plan/statement/ddl.rs

```diff
@@ -1717,6 +1729,18 @@ pub fn plan_create_view(
     } else {
         None
     };
+
+    // check for an object in the catalog with this same name
```

Comment:
Another tiny nit, we typically capitalize the first letter of a comment, though I can't find anything in our official guides explicitly recommending it.